src/aerie_cli/utils/sessions.py

```diff
@@ -15,6 +15,9 @@ def get_active_session_client():
         AerieClient
     """
     session = PersistentSessionManager.get_active_session()
+    hasura_secret = CommandContextManager.context.get_hasura_admin_secret()
+    if hasura_secret:
+        session.session.headers["x-hasura-admin-secret"] = hasura_secret
```

Comment:
I made the design decision to not allow users to configure an admin secret as part of a configurations. Instead, it's only currently settable via the command-line option or environment variable and only applies to the current command. IMO, specifying an admin secret should be treated like an escalation of privileges and therefore shouldn't be configurable as a default.

Reply:
Ah, my bad, I was thinking of how the client is only created once when AerieCLI is used in Python, but looking at it's usages I see that a new client is created per command. 

> Instead, it's only currently settable via the command-line option or environment variable and only applies to the current command. IMO, specifying an admin secret should be treated like an escalation of privileges and therefore shouldn't be configurable as a default.

That would explain my confusion in points 1 and 3 of my top-level review comment, since, as mentioned in point 2, it's currently applied globally forever when using the CLI option.

However, I feel that allowing it to be set per-command via an envvar in contradictory the desired design. Since the CLI _always_ picks up the envvar, setting it means that the user is always in Admin mode... for the deployment that envvar is compatible with. This not only creates usage problems when trying to work with multiple deployments, it goes against the desire for `specifying an admin secret should be treated like an escalation of privileges`.

Reply:
Yeah, that's what I'm thinking

Reply:
Sounds good, I removed the environment variable in a02e09d